.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# The global '*' pattern specifies code owners for all files in the repository.
+*       @tomassebestik @kumekay
```

Comment:
I would suggest adding more people in different locations. 

An even better approach is to create a team in the espressif organization and add people there, let's discuss it 1on1

Reply:
I'm still thinking about how to set this up in a meaningful and useful way. We need to set some code owners here to use the branch protection rule `Require review from Code Owners`, which seems to me to be a useful and logical rule. 

In simple projects, it probably doesn't make much sense to add more granularity. Usually, just one or two people are responsible for the entire repository, including docs and CI.


An individual discussion would be appreciated. Thanks!